### DISCUSSION TITLE
Non-Hierarchy Mind Maps

### DISCUSSION THREAD

**Discussion by MrDaisyBates** [freeplane/freeplane#1995] [General]
Hello everyone. I hope you're well. 

I used to struggle with organizing notes, files, everything really, because I always fell into the trap of making it a hierarchy (where one thing is always nested under something else). What I came to find is these issues went away when I switched to relational instead. 

Easy example:
I could have a node with "Red Items," and under it could be an apple, lipstick, and blood. These would be children.

The issue is, I could also have a node with "Makeup." Lipstick is clearly also a child under "Makeup."

However, lipstick is only one item (deserving of one node) yet exists 100% as makeup, but also 100% as "red items." Furthermore, in the right situation, it's also an emergency marker or writing instrument. 

When we use a hierarchy, we're attempting to organize information in a way helps think about it (categorization). However, instead of using a hierarchy, something more like a "tag" is beneficial. "Summon all things with tag X."

Anyways. I don't want to get into that. I just wanted to explain the issue. 

The question is, does Freeplane have the ability to operate in a non-hierarchy way? Can a node have multiple parents, for example and be organized as such?

https://github.com/freeplane/freeplane/discussions/1712

I found this discussion beforehand regarding "connectors," but I don't know that it exactly gives something two parents?

I like Mind Maps a lot, but the project I have in mind, I really like the ability to collapse and notate and things of that nature that come with Freeplane, but I need the flexibility of a flow chart, I think (because that isn't restricted to a hierarchy necessarily, though it could get messy).  

Thanks again, everyone!

**Comment by macmarrum** [comment]
In Freeplane, a node can have only one parent.

Connectors can be used to link a node to other nodes, which could be perceived as additional parents.

In Freeplane 1.12.x, the functionality of **tags** has been introduced. Tags can be categorized and searched for. Maybe you could give it a try and report back.

**Comment by dpolivaev** [comment]
You can also look at clones that allow the same nodes or subtrees appear at different places in the mins map.

**Comment by MrDaisyBates** [comment]
Thanks for the feedback, @macmarrum and @dpolivaev ! I will experiment with these things and see if I can get it the way I need it to be. Thanks again!
